tests/performance/conftest.py

```diff
@@ -0,0 +1,104 @@
+import collections
+import functools
+import logging
+import random
+from datetime import timedelta
+from functools import partial
+
+import databricks.sdk.core
+import pytest
+from databricks.sdk import AccountClient
+from databricks.sdk.core import Config
+from databricks.sdk.errors import NotFound
+from databricks.sdk.retries import retried
+
+from databricks.labs.ucx.mixins.fixtures import *  # noqa: F403
+
+logging.getLogger("tests").setLevel("DEBUG")
+logging.getLogger("databricks.labs.ucx").setLevel("DEBUG")
```

Comment:
If you're running this as a dedicated runner, it might make sense to route logs to a file as well.